tools/misc/get_image_metas.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Please add `openimage` prefix or suffix to indicate this script is only used for openimage

Reply:
@jshilong 
This script can get image meta on every dataset, not only on OpenImages datasets. Do we need to change the name?